src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Square extends Shape {
+    private double side;
+
+    public Square(String color, double side) {
+        super(Colors.valueOf(color));
+        this.side = Math.round(Math.abs(side));
```

Comment:
Rounding the side using `Math.round()` might not be necessary unless specified by the task requirements. Ensure this aligns with the intended functionality.